pages.es/common/more.md

```diff
@@ -0,0 +1,19 @@
+# more
+
+> Abre un archivo para lectura interactiva, permitiendo desplazar y buscar (Solamente en una dirección, adelante).
+
+- Abre un archivo:
+
+`more {{archivo}}`
+
+- Página abajo:
```

Comment:
I think we say be a little more verbose:
```suggestion
- Avanza hacia la siguiente página:
```

Reply:
@AlejandroCruz-0 what about this one?

Reply:
oh wow, i forgot about that one, oops. I'll fix it.